scripts/run_forge.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
```

Comment:
one thing I'm wondering: Maybe we should start organizing things more around features/functionalities, irrespective of the technology something it is implemented in.
Concretely I think it's a bit annoying that the forge rust cli is in a very different place than this script. Might be better to put this into the same directory as where the forge cli is imo. wdyt?

Reply:
That's a good point. We kind of already do that with everything other than `scripts` I think. Moving it under `testsuite`